### ISSUE TITLE
Resizing pattern at the start

### ISSUE THREAD

**Issue by NETMANSKY** [LMMS/lmms#4154]
Whether expect implement a resize of pattern at the start and not only at the end as now. The lack of this feature makes working with patterns in LMMS extremely uncomfortable :-(

**Comment by tresf** [comment]
Thanks.  This has been added to the master branch, available in a future release.  Sampletrack - resize from left

**Comment by tresf** [comment]
Just realized you may be talking about all patterns (BBEditor, Piano Roll, Automation).  Reopening. :)

**Comment by noahb01** [comment]
Agree! Would be especially useful for long buildups when you only want the last part in some circumstances. Currently you have to make a completely separate pattern.

**Comment by NETMANSKY** [comment]
Apparently not so simple :-)
